utils/vara-ui/src/components/checkbox/checkbox.tsx

```diff
@@ -1,28 +1,36 @@
 import { InputHTMLAttributes, forwardRef } from 'react';
 import cx from 'clsx';
 import styles from './checkbox.module.scss';
+import type { ICheckboxSizes, ICheckboxTypes } from './helpers.ts';
 
 type Props = InputHTMLAttributes<HTMLInputElement> & {
   label: string;
-  type?: 'switch';
+  type?: ICheckboxTypes;
+  checkboxSize?: ICheckboxSizes;
+  hasError?: boolean;
```

Comment:
```suggestion
  type?: 'checkbox' | 'switch';
  size?: 'default' | 'small';
  error?: string;
```

No need for extra file to handle types, it's small enough to be inlined, and let's stick to existing naming conventions